### PR TITLE
Improve style of reply bar

### PR DESCRIPTION
**Pull request by haileyok**

### CODE REVIEW

File: src/lib/custom-animations/ScaleOnPressIn.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import {Pressable, PressableProps} from 'react-native'
+import Animated, {
+  cancelAnimation,
+  useAnimatedStyle,
+  useSharedValue,
+  withTiming,
+} from 'react-native-reanimated'
+
+export function ScaleOnPressIn({
```
Comment (mozzius) [inline src/lib/custom-animations/ScaleOnPressIn.tsx:10]: Can we call it `TouchableScale` or `PressableScale` or something so it's clear it's a type of Touchable

Comment (haileyok) [reply src/lib/custom-animations/ScaleOnPressIn.tsx]: yerp

File: src/view/com/post-thread/PostThreadComposePrompt.tsx [hunk 2]
```diff
@@ -17,47 +20,57 @@ export function PostThreadComposePrompt({
...
+
+  const onPress = () => {
+    playHaptics('Light')
+    setTimeout(
+      () => {
+        onPressCompose()
+      },
+      isHapticsDisabled ? 0 : 75,
+    )
+  }
```
Comment (mozzius) [inline src/view/com/post-thread/PostThreadComposePrompt.tsx:37]: I added haptics to match the FAB compose button

**Review by mozzius — APPROVED** [review]
Looks great!

File: src/view/com/post-thread/PostThreadComposePrompt.tsx (hunk 2 above)
Comment (haileyok) [reply src/view/com/post-thread/PostThreadComposePrompt.tsx:37]: 🔥

**Pull request merged**